artifacts.go

```diff
@@ -58,9 +60,30 @@ func artifactWriter(f *os.File, c *cli.Context,
 
 func scripts(c *cli.Context) (*artifact.Scripts, error) {
 	scr := artifact.Scripts{}
-	for _, script := range c.StringSlice("script") {
-		if err := scr.Add(script); err != nil {
-			return nil, err
+	for _, scriptArg := range c.StringSlice("script") {
+		statInfo, err := os.Stat(scriptArg)
+		if err != nil {
+			return nil, errors.Wrapf(err, "can not stat script file: %s", scriptArg)
+		}
+
+		var scriptList []string
+		// Read either a directory, or add the script file directly.
+		if statInfo.IsDir() {
+			fileList, err := ioutil.ReadDir(scriptArg)
+			if err != nil {
+				return nil, errors.Wrapf(err, "can not list directory contents of: %s", scriptArg)
+			}
+			for _, statInfo = range fileList {
```

Comment:
I would use different name for `statInfo` here. Not a big deal, but easy to confuse with the one used above.